django_netjsonconfig/base/admin.py

```diff
@@ -200,6 +205,15 @@ def download_view(self, request, pk):
         return send_file(filename='{0}.tar.gz'.format(config.name),
                          contents=config_archive.getvalue())
 
+    def device_context_view(self, request, pk):
```

Comment:
after reviewing the code again, I realized we're editing a base class that is then inherited by the Template admin and VPN admin, this means the context API will generate different views and we're testing only one:

([show_urls is a django-extensions command](http://openwisp.io/docs/developer/hacking-openwisp-python-django.html#django-extensions))
```
./manage.py show_urls | grep context
/admin/django_netjsonconfig/device/<pk>/context\.json	django_netjsonconfig.base.admin.device_context_view	admin:django_netjsonconfig_device_context
/admin/django_netjsonconfig/template/<pk>/context\.json	django_netjsonconfig.base.admin.device_context_view	admin:django_netjsonconfig_template_context
/admin/django_netjsonconfig/vpn/<pk>/context\.json	django_netjsonconfig.base.admin.device_context_view	admin:django_netjsonconfig_vpn_context
```

Those views are now raising an exception and are not being tested.

`device_context_view` should be renamed to `context_view`.